pkg/backend/openshiftcluster/create.go

```diff
@@ -31,17 +31,49 @@ import (
 
 	"github.com/Azure/ARO-RP/pkg/api"
 	"github.com/Azure/ARO-RP/pkg/install"
+	"github.com/Azure/ARO-RP/pkg/util/pullsecret"
 	"github.com/Azure/ARO-RP/pkg/util/stringutils"
 	"github.com/Azure/ARO-RP/pkg/util/subnet"
 	"github.com/Azure/ARO-RP/pkg/util/version"
 )
 
+func getACRRegistryProfile(oc *api.OpenShiftCluster) *api.RegistryProfile {
```

Comment:
This is not for this PR, but I would like these kind of function to be connected to OpenShiftCluster object, so we could use them around the place, and not in the one `pkg`. Like:
```
func (o OpenShiftCluster) GetACRRegistryProfile() *RegistryProfile{
...
}
```
as now we are glueing these around the place, kinda helper functions.